man/systemd.link.xml

```diff
@@ -967,6 +967,17 @@
           <xi:include href="version-info.xml" xpointer="v232"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ReceivePacketSteeringCPUMask=</varname></term>
+        <listitem>
+          <para>Configures Receive Packet Steering (RPS) list of CPUs to which RPS may forward traffic.
+          RPS is disabled when it is zero (the default). Takes a list of CPUs and has the same syntax
```

Comment:
Here, `zero` sounds spurious for me, as `0` is a valid setting. Maybe, `Defaults to unset and RPS list is unchanged.` is better?